www/src/pages/docs/cli/integrations/token-listing.md

```diff
@@ -0,0 +1,525 @@
+Produce a Token Listing Format file for your design token build. The Token Listing Format is used by design token tool makers to understand the relationship between your source design tokens and your style files built in Terrazzo.
+
+## Setup
+
+Requires [Node.js 20 or later](https://nodejs.org) and [the CLI installed](https://terrazzo.app/docs/cli). With both installed, run:
+
+:::code-group
+
+```sh [npm]
+npm i -D @terrazzo/cli @terrazzo/plugin-token-listing
+```
+
+```sh [pnpm]
+pnpm i -D @terrazzo/cli @terrazzo/plugin-token-listing
+```
+
+```sh [bun]
+bun i -D @terrazzo/cli @terrazzo/plugin-token-listing
+```
+
+:::
+
+And add it to `terrazzo.config.js` under `plugins`:
+
+
+:::code-group
+
+```js [terrazzo.config.js]
+import { defineConfig } from "@terrazzo/cli";
+import css from "@terrazzo/plugin-css";
+import listing from "@terrazzo/plugin-token-listing";
+
+export default defineConfig({
+  outDir: "./tokens/",
+  plugins: [
+    // Include at least one plugin to build tokens
+    css({
+      filename: "tokens.css",
+    }),
+
+    // Configure the token listing
+    listing({
+      filename: 'tokens.listing.json',
+      // Pass mode information so documentation tools can generate mode selectors
+      modes: [
+        {
+          name: 'color-scheme',
+          values: ['light', 'dark'],
+          description: 'Color theme matching user device preferences',
+        },
+      ],
+      // Define platforms included in the Terrazzo build
+      platforms: {
+        css: {
+          description: 'Tokens built as CSS variables for the developers',
+          filter: '@terrazzo/plugin-css',
+          name: '@terrazzo/plugin-css',
+        },
+      },
+    }),
+  ],
+});
+```
+
+:::
+
+Lastly, run:
+
+```sh
+npx tz build
+```
+
+And you’ll see a `./tokens/tokens.listing.json` file generated in your project.
+
+
+## Usage
+
+This plugin creates a Token Listing file, which consolidates information about your source tokens and tokens built from other plugins. Token Listing is a community proposal that lists design tokens in a predictable, known format, to improve interoperability among design token tools. This format includes information about source tokens, modes, platforms where the tokens are built, and a mapping of token names between each platform.
+
+Without Token Listings, maintainers of design token tools must develop their own input format, and users of these tools must format their tokens for each individual tool they want to use. With Token Listings, tool makers have a predictable data input that's natively supported by Terrazzo and that can be shared across tools.
+
+Tools targeted by the Token Listing format include:
+* design token diffing algorithms
+* changelog and release note generators
+* design token review and QA tools
+* documentation websites
+* MCP servers
+* linting plugins
+* code migration tools
+* search indexes for design tokens
+
+
+Below are a few screenshots of what such tools can look like:
+
+// TODO
```

Comment:
- [ ] Missing the screenshots

Reply:
Not gonna include any for now to keep things simple and employer-conversation-free